src/port.js

```diff
@@ -232,6 +247,18 @@ export default class Port {
 				.resolve(handlerResult)
 				.then((val) => {
 					me.sendMessage(`res.${requestType}`, { id: w.id, val: val });
+				})
+				.catch((e) => {
```

Comment:
In theory couldn't `handler.apply(handler, w.args)` a few lines above also throw?